packages/lib/src/components/ThreeDS2/components/DeviceFingerprint/PrepareFingerprint3DS2.tsx

```diff
@@ -21,7 +21,12 @@ class PrepareFingerprint3DS2 extends Component<PrepareFingerprint3DS2Props, Prep
             };
         } else {
             this.state = { status: 'error' };
-            this.props.onError('Missing fingerprintToken parameter');
+            // TODO - confirm that we should do this, or is it possible to proceed to the challenge anyway?
```

Comment:
How do we verify this TODO?

Reply:
I'm awaiting an answer. But for now we can proceed (and just keep this here as a reminder)